sponsor-dapp/src/App.js

```diff
@@ -79,7 +80,7 @@ class App extends Component {
             if (this.props.params) {
               newParams = { ...this.props.params };
             } else {
-              newParams = { ...params };
+              newParams = { ...params, identifiers: { ...identifierConfig } };
```

Comment:
As discussed IRL, could we make the config.json file a .js file and import/append identifiers.json within that file?

Reply:
Makes a ton of sense to me, done.